config/default.js

```diff
@@ -72,7 +72,7 @@ module.exports = {
   // The version for the favicon.
   // This should be changed when a new favicon is pushed to the CDN to prevent
```

Comment:
Is this not something we can control from the CDN? I would be surprised if that were the case.

Reply:
We can invalidate the CDN cache, sure, but the file is still served with `Cache-Control: immutable` and `Cache-Control: max-age=315360000` to clients as well, so that would be pointless.

For other assets we simply change the filename automatically through the build process but since this one lives in addons-server and is referenced in addons-frontend, it's done manually. It changes so rarely that it's not a big deal.